src/mlpack/methods/ann/layer_names.hpp

```diff
@@ -372,6 +372,17 @@ class LayerNameVisitor : public boost::static_visitor<std::string>
     return "weightnorm";
   }
 
+  /*
```

Comment:
I think this should be` /**`

Reply:
The whole file has incorrect comment style (You can fix them here and once it gets merged it will be fixed for all). I have made the same changes in my PR.

Reply:
I can fix that in #2243 , since it is about fixing `layer_names.hpp`. It also didn't have header guard.